scss/atoms/typography/_typography.scss

```diff
@@ -21,40 +21,45 @@ body {
 }
 
 h1 {
-	font-size: em( 67px ); // 54px works nicely, as well
-	font-weight: 700;
-	line-height: 1;
+	@include heading($h1);
 }
 
 h2 {
-	font-size: em( 36px );
-	font-weight: 600;
-	line-height: 1;
+	@include heading($h2);
 }
 
 h3 {
-	font-size: em( 30px );
-	font-weight: 600;
-	line-height: 1;
+	@include heading($h3);
 }
 
 h4 {
-	font-size: em( 24px );
-	font-weight: 600;
-	line-height: 1;
+	@include heading($h4);
 }
 
 h5 {
-	font-size: em( 20px );
-	font-weight: 600;
-	line-height: 1;
+	@include heading($h5);
 }
 
 h6 {
-	font-size: em( 16px );
-	font-weight: 600;
-	line-height: 1;
-	text-transform: uppercase;
+	@include heading($h6);
+}
+
+p {
+	text-align: map-get($p,textAlignment);
+	margin: em(map-get($p,marginTopBottom)) em(map-get($p,marginLeftRight));
+	line-height: 1.5;
+	font-size: map-get($defaultFont, font-size);
```

Comment:
$defaultFont doesnot exist anymore. Its $font-default now. 
check https://github.com/jquery/css-chassis/blob/master/scss/variables/typography.js
  Rebase this branch on jquery/master to see. That will also remove conflicts.